client/app/shared/forms/distributed.js

```diff
@@ -108,6 +108,9 @@ var DistributedForm = BackboneForm.extend({
       });
     });
 
+    if(typeof window.formRendered == "function"){
+      window.formRendered();
```

Comment:
A better approach to hooking into this kind of functionality is in the individual view. The base ItemView triggers an 'editing' event on the individual view after the editor is rendered. See https://github.com/mediapublic/mediapublic/blob/develop/client/app/shared/views/itemview.js#L85.